android-components/build.gradle

```diff
@@ -320,8 +320,6 @@ detekt {
     config = files("$projectDir/config/detekt.yml")
     baseline = file("$projectDir/config/detekt-baseline.xml")
 
-    failFast = false
```

Comment:
The --fail-fast CLI flag (and failFast Gradle property) has been removed. It was deprecated since 1.16.x - https://github.com/detekt/detekt/pull/5290

See https://github.com/detekt/detekt/releases/tag/v1.22.0